notebooks/week2/05_log_and_register_fe_model.py

```
@@ -0,0 +1,280 @@
+
+# COMMAND ----------
+
+dbutils.library.restartPython()
+
+# COMMAND ----------
+
+import yaml
+import mlflow
+from mlflow.models import infer_signature
+
+from datetime import datetime
```

Ruff

Module level import not at top of file

(E402)